lib/bonny/axn.ex

```diff
@@ -211,8 +211,13 @@ defmodule Bonny.Axn do
 
   @doc """
   Registers a decending object to be applied.
-  Owner reference will be added automatically. unless disabled through
-  the option `omit_owner_ref`.
+  Owner reference will be added automatically unless disabled through
+  the option `omit_owner_ref` or if the resource is cluster-scoped.
+
+  **Note:** Cluster-scoped resources (resources without a namespace) will
+  automatically have owner references omitted, regardless of the
+  `omit_owner_ref` option, as they should never have owner references
+  according to Kubernetes best practices.
```

Comment:
Let's use the nice and shiny note blocks? Not 100% sure this syntax is correct :)

```suggestion
> #### No Owner References for Cluster-scoped Resources {: .info}
>
> Cluster-scoped resources (resources without a namespace) will
> automatically have owner references omitted, regardless of the
> `omit_owner_ref` option, as they should never have owner references
> according to Kubernetes best practices.
```

Reply:
yeah, looks good, you just had to indent once more :) 

<img width="1258" height="789" alt="Screenshot 2025-08-15 at 14 21 15" src="https://github.com/user-attachments/assets/cd6e3f73-cb09-4d18-9fc5-409d73628395" />